src/nightwatch/responsive-helpers.js

```diff
@@ -0,0 +1,39 @@
+const screenWidth = (browser) => {
+  const breakpoint = browser.currentTest.name.match(/\[@(.*?)\]/)[1];
```

Comment:
> const breakpoint = browser.breakpoint();

I took a look at updating to use this custom command, however, I think the current implementation is what we want. It seems custom commands are useful for pages interactions such as login steps, or repeated click events, etc. Writing a custom command to solely return the breakpoint value ends up returning the entire test object (which is what we started with). The alternative is to write a command with a callback function, but I have found if an error occurs, assertions within the callback neither pass nor fail.